src/clusterfuzz/_internal/bot/fuzzers/afl/launcher.py

```diff
@@ -642,9 +691,11 @@ def do_offline_mutations(self):
         remove_path(input_path)
 
   def generate_afl_args(self,
-                        afl_input=None,
-                        afl_output=None,
-                        mem_limit=constants.MAX_MEMORY_LIMIT):
+      afl_input=None,
+      afl_output=None,
+      target_path=None,
+      additional_args=[],
```

Comment:
Use `None` instead of a list here: https://google.github.io/styleguide/pyguide.html#2124-decision